go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/alecthomas/kong v0.8.0
 	github.com/bufbuild/buf v1.25.1
-	github.com/crossplane/crossplane-runtime v0.20.0
+	github.com/crossplane/crossplane-runtime v0.20.1-0.20230804074918-89472a2578ce
```

Comment:
I'd rather cut a release of crossplane-runtime. What about backporting? what versions are release branches using atm?

Reply:
`release-1.13` and `release-1.12` are using crossplane-runtime `1.20.*`, and `release-1.11` is using `1.19.*`. I'm ok with the new release creation.

Reply:
Here is the backport PR -> https://github.com/crossplane/crossplane-runtime/pull/497